src/electron/main/main.ts

```diff
@@ -15,7 +15,7 @@ const createWindow = () => {
     });
 
     mainWindow
-        .loadFile(path.resolve(__dirname, '../electron/views/device-connect-view/deviceConnectView.html'))
```

Comment:
any particular reason why we didn't move `index.html` to be directly under `views` folder?

Reply:
I did and I forgot to push the commit before opening the PR